dashboard/package.json

```diff
@@ -20,12 +20,18 @@
   "bugs": {
     "url": "https://github.com/ucberkeley/sis-qa-test-auto/issues"
   },
-  "devDependencies": {
+  "dependencies": {
+    "browserify": "~13.0.0",
```

Comment:
@mDibyo is there a reason we put them in different sections - `devDependencies` and `dependencies`? Could we combine them instead?